spec/controllers/api/v1/concepts_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe Api::V1::ConceptsController, type: :controller do
+
+  context 'POST #create' do
+    let!(:user) {FactoryGirl.create(:admin)}
+    let!(:token) { double :acceptable? => true, :resource_owner_id => user.id }
+    let!(:concept_name) {'Concept1'}
+    let!(:parent_concept) {FactoryGirl.create(:concept)}
+
+    def subject
+      post :create, {name: concept_name, parent_uid: parent_concept.uid}
+    end
+
+    context 'OAUTH' do
+      it_behaves_like 'protected endpoint'
+    end
+
```

Comment:
@marcellosachs You can leave out the `context` here. You could also keep it, but it doesn't really provide any extra info. I typically save the extra nesting for specs requiring additional setup.